packages/p2p-v2/src/features/my-profile/screens/MyProfileStats/MyProfileStats.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { useActiveAccount } from '@deriv/api';
+import { useAdvertiserStats } from '../../../../hooks';
```

Comment:
Can't we add absolute import?

Reply:
There's issue with declaring the relative imports, I can do it in another PR, this PR can establish the initial project structure

Reply:
ok then thanks @adrienne-deriv 👍